bears/css/CSSAutoPrefixBear.py

```diff
@@ -0,0 +1,16 @@
+from coalib.bearlib.abstractions.Lint import Lint
+from coalib.bears.LocalBear import LocalBear
+
+
+class CSSAutoPrefixBear(Lint, LocalBear):
+    executable = "postcss"
```

Comment:
It looks as if `postcss` is the executable and `autoprefixer` is a plugin for it.

Should we have `PostCSSBear` and add an argument for this then ?